### DISCUSSION TITLE
[Feature Request] - Add an option to automatically enable "Off The Radar" when "No clip" is enabled.

### DISCUSSION THREAD

**Discussion by BUZZARDGTA** [YimMenu/YimMenu#2797] [Ideas]
This would prevents players currently recording their screen be able to reports us as cheating.
Well technically you cannot be Off The Radar for only a few seconds? So in the end idk if the idea is much useful here.

**Comment by xesdoog** [comment]
Why don't you just enable it yourself? I'm pretty sure both of those options are right next to each other.
